e2e/cypress-utils.js

```diff
@@ -86,3 +83,26 @@ export const openAsMobile = (url) => {
     }
   })
 }
+
+export const setupEmbedOnMobile = () => {
+  if (Cypress.isBrowser('chrome')) { // inside of an iframe is testable only in Chrome
+    getIframeBody(IFRAME).find('[data-qa="start-button"]').should('have.text', 'Start').click()
+
+    cy.wait(1500).then(() => {
```

Comment:
Just a small comment. Maybe we could add these waiting times in a const variable so it is easy to adjust them at once if needed.

Reply:
makes sense, in some cases might not be needed but the main issue is the loading time of the iframe. Let me me have a look later and I'll move them in some variables

Reply:
fixed now thanks!